package.json

```diff
@@ -126,8 +126,8 @@
     "@storybook/manager-webpack5": "6.5.15",
     "@storybook/react": "6.5.15",
     "@storybook/theming": "6.5.15",
-    "@swc-node/register": "1.6.1",
```

Comment:
removing for now as swc-node is incompatible with swc/helpers 0.4 - thus making nx explode. as a fallback nx will use ts-node which is our only option for now.